CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 * `scrollToWithOptions` method to `window` (#26)
 * `HtmlDocument` methods `hasFocus` and `activeElement` moved to `Document` (#41)
   * They're still available on `HtmlDocument` thanks to the `include` structure
+* `StaticRange` bindings (#47)
```

Comment:
```suggestion
* `StaticRange` bindings (#48)
```

Reply:
Oh I thought these should reference tickets not PRs.

Reply:
It's a little confusing, I agree; I choose PRs. I'm hoping to automate this at some point.